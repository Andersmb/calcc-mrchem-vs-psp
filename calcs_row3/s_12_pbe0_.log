Starting job 5688136 on c2-[33,38],c3-6,c5-[33,40] at Fri May 20 08:26:22 CEST 2022

Copying s_12_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying s_12_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688136      s_12_pbe0_        180                                             00:46:24      0:0 
5688136.bat+      batch         36        1   00:00:03          0   00:00:03   00:46:24      0:0 
5688136.ext+     extern        180        5   00:00:00          0   00:00:00   00:46:24      0:0 
5688136.0      mrchem.x        180       30   00:45:40         29   02:09:00   00:45:50      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688136                                                                          
5688136.bat+     19268K          0     19268K        0              0          0 
5688136.ext+          0          0          0        0              0          0 
5688136.0     16663816K         29 6673514086        0             16          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688136                                                                                               
5688136.bat+       19.41M               0         19.41M        0.07M                0          0.07M 
5688136.ext+        0.00M               0          0.00M            0                0              0 
5688136.0           6.75M               0          6.73M      157.17M               10         87.88M 

Job 5688136 completed at Fri May 20 09:12:46 CEST 2022
