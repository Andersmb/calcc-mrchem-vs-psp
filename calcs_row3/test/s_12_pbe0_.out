                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000038        0.000000
    1       O            :        0.000000        2.678724        0.000000
    2       O            :        0.000000       -1.339400        2.319828
    3       O            :        0.000000       -1.339400       -2.319828
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              20
 Beta electrons          :                                              20
 Total electrons         :                                              40
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000001801e+00
    1    1    a          :                              9.999999389257e-01
    2    1    a          :                              9.999996250282e-01
    3    1    a          :                              9.999999855800e-01
    4    1    a          :                              9.999999965135e-01
    5    1    a          :                              1.000000002734e+00
    6    1    a          :                              9.999999994941e-01
    7    1    a          :                              1.000000000172e+00
    8    1    a          :                              9.999994185809e-01
    9    1    a          :                              9.999999388117e-01
   10    1    a          :                              1.000000042032e+00
   11    1    a          :                              9.999969019369e-01
   12    1    a          :                              1.000000944543e+00
   13    1    a          :                              9.999989508326e-01
   14    1    a          :                              1.000000156746e+00
   15    1    a          :                              1.000002122187e+00
   16    1    a          :                              1.000000590351e+00
   17    1    a          :                              1.000004366544e+00
   18    1    a          :                              1.000000008903e+00
   19    1    a          :                              9.999997990407e-01
   20    1    b          :                              1.000000001801e+00
   21    1    b          :                              1.000000122648e+00
   22    1    b          :                              1.000000102215e+00
   23    1    b          :                              1.000000001649e+00
   24    1    b          :                              9.999999978022e-01
   25    1    b          :                              1.000000002692e+00
   26    1    b          :                              9.999999994450e-01
   27    1    b          :                              1.000000000173e+00
   28    1    b          :                              9.999994152591e-01
   29    1    b          :                              1.000000014136e+00
   30    1    b          :                              1.000000048810e+00
   31    1    b          :                              9.999969061781e-01
   32    1    b          :                              1.000000962612e+00
   33    1    b          :                              9.999988752333e-01
   34    1    b          :                              1.000000156449e+00
   35    1    b          :                              1.000002207777e+00
   36    1    b          :                              1.000000590887e+00
   37    1    b          :                              1.000004399527e+00
   38    1    b          :                              1.000000002076e+00
   39    1    b          :                              9.999997996535e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                618.804929291108
 E-N energy              :            (au)              -1836.683596297495
 Coulomb energy          :            (au)                464.986403949850
 Exchange energy         :            (au)                -12.524414925695
 X-C energy              :            (au)                -38.942499126406
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                184.733676503914
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -804.359177108639
 Nuclear energy          :            (au)                184.733676503914
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.196255006047e+02
                         :      (kcal/mol)             -3.888208688633e+05
                         :        (kJ/mol)             -1.626826516380e+06
                         :            (eV)             -1.686086808375e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.244890843341
    1    1    a          :            (au)                -18.901748700409
    2    1    a          :            (au)                -18.879086660488
    3    1    a          :            (au)                -18.876950768782
    4    1    a          :            (au)                 -8.675917337278
    5    1    a          :            (au)                 -6.624182977478
    6    1    a          :            (au)                 -6.622323961401
    7    1    a          :            (au)                 -6.622186701567
    8    1    a          :            (au)                 -1.359380729208
    9    1    a          :            (au)                 -1.142347948430
   10    1    a          :            (au)                 -1.136211358587
   11    1    a          :            (au)                 -0.614372952618
   12    1    a          :            (au)                 -0.531430251760
   13    1    a          :            (au)                 -0.459471079471
   14    1    a          :            (au)                 -0.457992817019
   15    1    a          :            (au)                 -0.310135648264
   16    1    a          :            (au)                 -0.308978073020
   17    1    a          :            (au)                 -0.295362925617
   18    1    a          :            (au)                 -0.287732986810
   19    1    a          :            (au)                 -0.248636894976
   20    1    b          :            (au)                -89.244891985418
   21    1    b          :            (au)                -18.901747732890
   22    1    b          :            (au)                -18.879091849427
   23    1    b          :            (au)                -18.876949778591
   24    1    b          :            (au)                 -8.675917300399
   25    1    b          :            (au)                 -6.624183003719
   26    1    b          :            (au)                 -6.622324058858
   27    1    b          :            (au)                 -6.622186636177
   28    1    b          :            (au)                 -1.359378642347
   29    1    b          :            (au)                 -1.142402295566
   30    1    b          :            (au)                 -1.136207916900
   31    1    b          :            (au)                 -0.614373126066
   32    1    b          :            (au)                 -0.531432402381
   33    1    b          :            (au)                 -0.459478819079
   34    1    b          :            (au)                 -0.457985697047
   35    1    b          :            (au)                 -0.310133593407
   36    1    b          :            (au)                 -0.308983853821
   37    1    b          :            (au)                 -0.295361550787
   38    1    b          :            (au)                 -0.287732638078
   39    1    b          :            (au)                 -0.248636908350
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -363.198741405832
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          6.324555e+00        -619.625500604724         -6.196255e+02
    1          5.202370e-01        -623.326173075988         -3.700672e+00
    2          2.201317e-01        -623.519786258749         -1.936132e-01
    3          1.101762e-01        -623.565539348403         -4.575309e-02
    4          6.772442e-02        -623.582176500970         -1.663715e-02
    5          8.800868e-02        -623.598277975083         -1.610147e-02
    6          1.972389e-02        -623.600035682450         -1.757707e-03
    7          2.515262e-02        -623.601438634262         -1.402952e-03
    8          1.052952e-02        -623.601626343244         -1.877090e-04
    9          8.638203e-03        -623.601720778007         -9.443476e-05
   10          2.925040e-03        -623.601750303011         -2.952500e-05
   11          2.454658e-03        -623.601762798844         -1.249583e-05
   12          8.044308e-04        -623.601764900779         -2.101936e-06
   13          6.935120e-04        -623.601765939867         -1.039088e-06
   14          4.413523e-04        -623.601766371725         -4.318576e-07
   15          2.866301e-04        -623.601766560118         -1.883934e-07
   16          1.968429e-04        -623.601766620984         -6.086600e-08
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000038        0.000000
    1       O            :        0.000000        2.678724        0.000000
    2       O            :        0.000000       -1.339400        2.319828
    3       O            :        0.000000       -1.339400       -2.319828
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                622.335038330938
 E-N energy              :            (au)              -1847.785703148633
 Coulomb energy          :            (au)                468.314498890813
 Exchange energy         :            (au)                -12.440018404868
 X-C energy              :            (au)                -38.759258793149
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                184.733676503914
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -808.335443124899
 Nuclear energy          :            (au)                184.733676503914
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.236017666210e+02
                         :      (kcal/mol)             -3.913160134398e+05
                         :        (kJ/mol)             -1.637266201295e+06
                         :            (eV)             -1.696906778938e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.274140594496
    1    1    a          :            (au)                -19.290640221746
    2    1    a          :            (au)                -19.290631170956
    3    1    a          :            (au)                -19.290614442178
    4    1    a          :            (au)                 -8.229666388650
    5    1    a          :            (au)                 -6.169527596918
    6    1    a          :            (au)                 -6.163335845841
    7    1    a          :            (au)                 -6.163335724671
    8    1    a          :            (au)                 -1.254645334680
    9    1    a          :            (au)                 -1.122737193675
   10    1    a          :            (au)                 -1.122719936456
   11    1    a          :            (au)                 -0.654481790243
   12    1    a          :            (au)                 -0.581344033934
   13    1    a          :            (au)                 -0.557363584113
   14    1    a          :            (au)                 -0.557359918883
   15    1    a          :            (au)                 -0.452305768776
   16    1    a          :            (au)                 -0.452303637803
   17    1    a          :            (au)                 -0.426109875136
   18    1    a          :            (au)                 -0.426097966901
   19    1    a          :            (au)                 -0.370661718757
   20    1    b          :            (au)                -89.274140593428
   21    1    b          :            (au)                -19.290640222630
   22    1    b          :            (au)                -19.290631175273
   23    1    b          :            (au)                -19.290614441408
   24    1    b          :            (au)                 -8.229666387155
   25    1    b          :            (au)                 -6.169527593542
   26    1    b          :            (au)                 -6.163335845006
   27    1    b          :            (au)                 -6.163335723750
   28    1    b          :            (au)                 -1.254645334225
   29    1    b          :            (au)                 -1.122737201625
   30    1    b          :            (au)                 -1.122719933340
   31    1    b          :            (au)                 -0.654481790801
   32    1    b          :            (au)                 -0.581344029621
   33    1    b          :            (au)                 -0.557363581095
   34    1    b          :            (au)                 -0.557359927164
   35    1    b          :            (au)                 -0.452305770095
   36    1    b          :            (au)                 -0.452303639694
   37    1    b          :            (au)                 -0.426109888671
   38    1    b          :            (au)                 -0.426097966478
   39    1    b          :            (au)                 -0.370661719570
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -363.700045509386
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000        0.000069        0.000079
 Magnitude               :            (au)                        0.000105
                         :         (Debye)                        0.000267
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000       -0.000001        0.000000
 Magnitude               :            (au)                        0.000001
                         :         (Debye)                        0.000003
---------------------------------------------------------------------------
 Total vector            :       -0.000000        0.000068        0.000079
 Magnitude               :            (au)                        0.000104
                         :         (Debye)                        0.000265
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 45m 42s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
